blockdag/dag.go

```diff
@@ -1635,10 +1641,10 @@ func (dag *BlockDAG) blueBlocksBetween(lowHash, highHash *daghash.Hash, maxEntri
 	}
 
 	// Collect every node in highNode's past (including itself) but
-	// NOT in the lowNode's past (excluding itself) into an UpHeap
+	// NOT in the lowNode's past (excluding itself) into an upHeap
```

Comment:
it's still weird to write upHeap in camel case if it doesn't appear in the code as a single word